content/doc/book/pipeline/getting-started.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 layout: section
+title: Getting Started with Pipeline
```

Comment:
You used `title:` here, but no title elsewhere.   What are the criteria for which one to use?